chromium_src/chrome/browser/ui/toolbar/toolbar_actions_model_factory.cc

```diff
@@ -0,0 +1,6 @@
+#include "chrome/browser/ui/toolbar/toolbar_actions_model.h"
+#include "brave/browser/ui/toolbar/brave_toolbar_actions_model.h"
+
+#define ToolbarActionsModel BraveToolbarActionsModel
```

Comment:
Always #undef after using #define and doing the include.

Reply:
This can't be used because the base class has a static function with that return type in the file you're directly patching.